LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2023 Northern.tech AS
+Copyright 2024 Northern.tech AS
```

Comment:
This I already fixed in https://github.com/mendersoftware/meta-mender/pull/2048 - sorry that I didn't see your PR before.

Reply:
ok thanks